iFixit/react-commerce
# Add store home page browse section

closes #856 

## QA

1. Open Vercel preview
2. Verify that the section is implemented as specified in #856 
3. 

## Post merge

This PR adds a new content type so after having merged this we need to migrate the Strapi live server

cc @sterlinghirsh
